src/lib/storage/dictionary_compression.hpp

```diff
@@ -6,12 +6,19 @@
 #include "all_type_variant.hpp"
 #include "types.hpp"
 
+#include "storage/encoding_type.hpp"
+
 namespace opossum {
 
 class BaseColumn;
 class Chunk;
 class Table;
 
+/**
+ * @brief Compresses parts of or an entire table using one encoding type
+ *
+ * @deprecated Use ChunkEncoder interface instead!
```

Comment:
Ah, I was quite confused when I saw the cpp file first. Rename to `deprecated_dictionary_compression`?

Reply:
renamed!